mod/ping.php

```diff
@@ -202,6 +222,13 @@ function xmlize($n){
 				<net>$network</net>
 				<home>$home</home>\r\n";
 		if ($register!=0) echo "<register>$register</register>";
+		if ( count($network_group) ) {
+			echo '<groups uid="' . intval(local_user()) . '">';
```

Comment:
not an issue, but uid is uneeded, as `/ping` page return values only for the current logged in user (you can't ping other users)